apps/charterafrica/src/components/Tool/Tool.js

```diff
@@ -62,18 +62,20 @@ const Tool = React.forwardRef(function Tool(props, ref) {
                 <RichTypography color="neutral.dark" variant="h2SemiBold">
                   {name}
                 </RichTypography>
-                <Link href={link.href}>
-                  <SvgIcon
-                    inheritViewBox
-                    component={ExternalLink}
-                    sx={{
-                      color: "text.primary",
-                      fill: "none",
-                      height: 32,
-                      width: 32,
-                    }}
-                  />
-                </Link>
+                {externalLink?.href ? (
+                  <Link href={externalLink?.href}>
```

Comment:
```suggestion
                  <Link href={externalLink.href}>
```

We've just confirmed that `externalLink.href` exists on the previous line.